scripts/ci/gke.sh

```diff
@@ -22,13 +22,10 @@ assign_env_variables() {
     info "Assigning environment variables for later steps"
 
     if [[ "$#" -lt 1 ]]; then
```

Comment:
Might be good to fail early rather than silently ignore args.
```suggestion
    if [[ "$#" -ne 1 ]]; then
```